### PR TITLE
Added help section #431

### LINKED ISSUE #431: Homepage improvements : Help section
We want to start adding more body to our homepage. One of the things is to add a "help section".
This story relates to adding that particular section.

As seen in the screenshot, we want : 

- [ ] A separator to show the beginning of the new section (1px grey line)
- [ ] Image illustrating the help (see attached image)
- [ ] All required text as seen in the screenshot
- [ ] Working links to https://help.fundrequest.io  and  mailto link to support@fundrequest.io

### Be aware

- Please use the existing colors, fonts, fonts-sizes,....
- This story is **ONLY** for the help section 

#### Screenshot of what is expected (seen bottom section)
image

#### Image to use for the help section
image-help

### PR DESCRIPTION
**Pull request by rsercano**
fixes #431 
https://fundrequest.io/requests/30

### CODE REVIEW

File: tweb/src/main/frontend/src/scss/fundrequest/_main-page-help.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.help-text {
+  p {
```
Comment (Simsteve7) [inline tweb/src/main/frontend/src/scss/fundrequest/_main-page-help.scss]: no use of pixels

File: tweb/src/main/resources/templates/index.html [hunk 2]
```diff
@@ -83,7 +84,37 @@ <h1 class="home-title">
                 </div>
             </div>
         </div>
+        <div class="container my-5 px-5">
+            <div class="row">
+                <div class="col-12 col-md-3 col-lg-3 offset-md-2">
+                    <img style="width: 87%;"
+                         src="https://user-images.githubusercontent.com/6882032/41536582-52e43a30-7306-11e8-92df-2fe6502ad760.png"/>
```
Comment (nico-ptrs) [inline tweb/src/main/resources/templates/index.html]: The image should be put in `tweb/src/main/frontend/src/img/` and then be referenced like `th:src="@{/assets/img/<image-name>.png}"` in the tag

File: tweb/src/main/resources/templates/index.html [hunk 3]
```diff
@@ -13,31 +13,32 @@
                 <div class="col-12 col-md-6 col-lg-5">
                     <h1 class="home-title">
                         Harness the power
-                        <br />of Open Source developers
+                        <br/>of Open Source developers
```
Comment (Simsteve7) [inline tweb/src/main/resources/templates/index.html]: revert unneeded changes

File: tweb/src/main/resources/templates/index.html [hunk 4]
```diff
@@ -13,31 +13,32 @@
...
                     </h1>
                     <div class="d-block d-md-none my-2 mx-5">
                         <img th:src="@{/assets/img/headerimg.png}"
-                             alt="A deventralized marketplace for open source collaboration" />
```
Comment (Simsteve7) [inline tweb/src/main/resources/templates/index.html]: revert unneeded changes

File: tweb/src/main/resources/templates/index.html [hunk 5]
```diff
@@ -13,31 +13,32 @@
...
+                             alt="A deventralized marketplace for open source collaboration"/>
                     </div>
                     <div class="home-description">
                         <p>
                             Easy and secure way to request
-                            <br />development bugfixes, security audits, ...
-                            <br />to the Open Source developer community.
```
Comment (Simsteve7) [inline tweb/src/main/resources/templates/index.html]: revert unneeded changes

File: tweb/src/main/resources/templates/index.html [hunk 6]
```diff
@@ -13,31 +13,32 @@
...
+                            <br/>development bugfixes, security audits, ...
+                            <br/>to the Open Source developer community.
                         </p>
                     </div>
                     <div class="d-flex align-items-center">
                         <a class="btn btn-secondary btn-lg btn--flat" th:href="@{/fund/github}">
                             Get started
                         </a>
-                        <a class="btn btn-link btn-lg text-white" th:href="@{https://help.fundrequest.io/how-does-it-work}">
```
Comment (Simsteve7) [inline tweb/src/main/resources/templates/index.html]: revert unneeded changes

File: tweb/src/main/resources/templates/index.html [hunk 7]
```diff
@@ -13,31 +13,32 @@
...
+                        <a class="btn btn-link btn-lg text-white"
+                           th:href="@{https://help.fundrequest.io/how-does-it-work}">
                             How it works...
                         </a>
                     </div>
                 </div>
                 <div class="col-12 col-md-5 offset-md-1 col-lg-4 d-none d-md-flex justify-content-cente">
                     <img class="align-middle" th:src="@{/assets/img/headerimg.png}"
-                         alt="A deventralized marketplace for open source collaboration" />
```
Comment (Simsteve7) [inline tweb/src/main/resources/templates/index.html]: revert unneeded changes

File: tweb/src/main/resources/templates/index.html [hunk 8]
```diff
@@ -83,7 +84,37 @@ <h1 class="home-title">
                 </div>
             </div>
         </div>
+        <div class="container my-5 px-5">
+            <div class="row">
+                <div class="col-12 col-md-3 col-lg-3 offset-md-2">
+                    <img style="width: 87%;"
```
Comment (Simsteve7) [inline tweb/src/main/resources/templates/index.html]: no inline styling

File: tweb/src/main/resources/templates/index.html [hunk 9]
```diff
@@ -83,7 +84,37 @@ <h1 class="home-title">
...
+                         src="https://user-images.githubusercontent.com/6882032/41536582-52e43a30-7306-11e8-92df-2fe6502ad760.png"/>
+                </div>
+                <div class="col-12 col-md-7 col-lg-7 help-text">
+                    <h2 class="help-text__header">
+                        Have a question or need some help?
+                        <br/>
```
Comment (Simsteve7) [inline tweb/src/main/resources/templates/index.html]: No <br/> for spacing, only for breaking text when really needed.

File: tweb/src/main/resources/templates/index.html [hunk 10]
```diff
@@ -83,7 +84,37 @@ <h1 class="home-title">
...
+                        </li>
+                        <li><i class="fa-li fa fa-comment-alt"></i> Have a chat with us</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
     </section>
 </th:block>
 
-</html>
```
Comment (Simsteve7) [inline tweb/src/main/resources/templates/index.html]: revert unneeded changes

File: tweb/src/main/resources/templates/index.html [hunk 11]
```diff
@@ -83,7 +84,37 @@ <h1 class="home-title">
                 </div>
             </div>
         </div>
+        <div class="container my-5 px-5">
```
Comment (nico-ptrs) [inline tweb/src/main/resources/templates/index.html]: One of the requirements of this issue is:
> A separator to show the beginning of the new section (1px grey line)

this still needs to be added

File: tweb/src/main/resources/templates/index.html (hunk 2 above)
Comment (nico-ptrs) [reply tweb/src/main/resources/templates/index.html]: Also the white background of the image should be transparant, we'll supply you with a new image to use

File: tweb/src/main/resources/templates/index.html (hunk 11 above)
Comment (rsercano) [reply tweb/src/main/resources/templates/index.html]: Consider it done, and please share the transparant image with me :)

File: tweb/src/main/resources/templates/index.html (hunk 2 above)
Comment (nico-ptrs) [reply tweb/src/main/resources/templates/index.html]: @rsercano we've updated the image in the issue, so you can use that

**Comment by rsercano** [comment]
Cany you check again please ?

File: tweb/src/main/frontend/src/scss/fundrequest/_main-page-help.scss (hunk 1 above)
Comment (rsercano) [reply tweb/src/main/frontend/src/scss/fundrequest/_main-page-help.scss]: done

File: tweb/src/main/resources/templates/index.html (hunk 4 above)
Comment (rsercano) [reply tweb/src/main/resources/templates/index.html]: done

File: tweb/src/main/resources/templates/index.html (hunk 3 above)
Comment (rsercano) [reply tweb/src/main/resources/templates/index.html]: done

File: tweb/src/main/resources/templates/index.html (hunk 5 above)
Comment (rsercano) [reply tweb/src/main/resources/templates/index.html]: done

File: tweb/src/main/resources/templates/index.html (hunk 7 above)
Comment (rsercano) [reply tweb/src/main/resources/templates/index.html]: done

File: tweb/src/main/resources/templates/index.html (hunk 6 above)
Comment (rsercano) [reply tweb/src/main/resources/templates/index.html]: done

File: tweb/src/main/resources/templates/index.html (hunk 11 above)
Comment (rsercano) [reply tweb/src/main/resources/templates/index.html]: done

File: tweb/src/main/resources/templates/index.html (hunk 8 above)
Comment (rsercano) [reply tweb/src/main/resources/templates/index.html]: done

File: tweb/src/main/resources/templates/index.html (hunk 2 above)
Comment (rsercano) [reply tweb/src/main/resources/templates/index.html]: done

File: tweb/src/main/resources/templates/index.html (hunk 9 above)
Comment (rsercano) [reply tweb/src/main/resources/templates/index.html]: done

File: tweb/src/main/resources/templates/index.html (hunk 10 above)
Comment (rsercano) [reply tweb/src/main/resources/templates/index.html]: done

**Pull request merged**